README.md

```diff
@@ -4,60 +4,114 @@ This repository contains examples showing how to consume the PowerDMS File Picke
 
-### `initializePowerDmsFilePicker`
-`initializePowerDmsFilePicker` accepts a configuration object as a single parameter. This object has the following structure:
+<br />
 
-#### JavaScript
 
-```javascript
-{
-  // String, required
-  // The API key given to your organization from PowerDMS.
-  apiKey,
+## Common Integration Scenarios
+The File Picker returns metadata about selected files. You may need to orchestrate additional API calls to achieve the intended integration experience. This section provides examples for some common scenarios.     
 
-  // Function, required
-  // A callback that is called when a user makes a selection.
-  onSelection,
+<br />
 
-  // Number, optional
-  // The desired width of the file picker, in pixels.
-  // Restricted from 566 to 1051.
-  width,
+### Downloading the Selected Document
+The File Picker selection object contains all of the information necessary to download the current revision of the selected file. To do so, call the `/documents/{documentId}/revisions/{revisionId}/content` endpoint using the `documentId` and `revisionId` from the selected file. For convenience, the correct link for the selected document is passed as the `contentUrl` property on the selection result. See the specific endpoint documentation (link tbd) for details.   
 
-  // Number, optional
-  // The desired height of the file picker, in pixels.
-  // Restricted from 350 o 650.
-  height
+<br />
+
+### Linking to the Selected Document
+You can construct a link to the selected document that allows an authenticated user to view the latest version of the selected document. The format of the link is `https://powerdms.com/link/{siteKey}/document/?id={documentId}` where:
+- `{siteKey}` is the PowerDMS Site Key that a user logging into PowerDMS specifies to identify the correct site to log into. This is not a part of the File Picker response and will need to be prompted for in a different way. 
+- `{documentId}` is the id of the selected document. This maps to the `documentId` property on the selected file response object. 
+
+For example, to link to document 20 in site AcmeCorp, you'll construct a link `https://powerdms.com/link/AcmeCorp/document/?id=8`.
+
+<br />
+
+### Active Revision on a Specific Date
+To link to the previous version of the selected document that was active on a certain, date, you will need to: 
+
+Call `POST /documents/past-published-revisions`. In the message body, specify the date at which you want the active revision, and the ids of the documents whose active revisions you want at the specified date. For example, to find the revisions of documents 10, 15, and 77 that were active on October 5th, 2019, you would make a call to `POST /documents/past-published-revisions` with a request body of 
+```json
+{
+   "targetDate": "2019-10-05T04:00:00.000Z",

```

Comment:
🔧 if the date does not have a timezone offset, it defaults to the timezone in site settings

Reply:
Added a note describing the default timezone behavior.